soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -313,6 +313,69 @@ static bool EntranceHandler(std::shared_ptr<Ship::Console> Console, const std::v
     gSaveContext.nextTransition = 11;
 }
 
+static bool VoidHandler(std::shared_ptr<Ship::Console> Console, const std::vector<std::string>& args) {
+    if (gGlobalCtx) {
```

Comment:
```suggestion
    if (gGlobalCtx != nullptr) {
```
To follow our conventions.